src/stylesheets/elements/_inputs.scss

```diff
@@ -1,3 +1,21 @@
+$input-line-height: 1.3;
+$input-border-width: 1px;
+$input-padding-v: 1rem;
+$input-padding-h: 0.7em;
+
```

Comment:
I tend to follow this guideline for adding in new variables (http://sass-guidelin.es/#variables) in which both criteria should be met:

- the value is repeated at least twice;
- the value is likely to be updated at least once;

Are both of these being met?